devel/addon/projectcontour/contour-gateway.yaml

```diff
@@ -5275,3713 +5276,3 @@ spec:
         runAsNonRoot: true
         runAsUser: 65534
         runAsGroup: 65534
-
----
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
These are being applied separately from https://github.com/cert-manager/cert-manager/tree/master/devel/addon/projectcontour in Bazel and via https://github.com/cert-manager/cert-manager/blob/master/make/e2e-setup.mk#L211 in make